README.md

```diff
@@ -88,13 +88,20 @@ Add the application key provided to you by Placed in the application tag of your
 <meta-data android:name="placed_app_key" android:value="YOUR_APP_KEY" />
 ```
 
-#### Prompting for location permission
+#### Opt-in Dialogue and Prompting for Location Permission
 
-If you don't already prompt for fine location permission, you should do so before registering the user with the Placed SDK.
+As outlined in Section 5 of the [Placed Affiliate Agreement](https://affiliate.placed.com/placed-affiliate-agreement/), you must satisfy two requirements prior to registering a user with the Placed SDK:  
+1. *Gather Express Consent for User Data Collection via Opt-in Dialog*  
+In addition having a legally compliant privacy policy describing Placed’s collection of location and device information, you must include a discrete opt-in dialog which gathers express consent for data collection. This dialog appears prior to the fine location permission prompt, and includes:
+    - The language: “*Aggregated device data, including location and apps, is measured for the purposes of market research by Placed, Inc.*”
+    - Links to the Placed [Terms of Service](https://www.placed.com/terms-of-service) and [Privacy Policy](https://www.placed.com/privacy-policy)
+    - Buttons to “Accept” or “Cancel”
 
-For an example, see the [main activity in the sample app](https://github.com/placed/android-placed-sdk/blob/master/SampleApp/app/src/main/java/com/placed/android/sampleapp/MainActivity.java).
 
-Notice in this case the fine location permission is requested before registering the user and acceptance triggers app registion as described below.
+1. *Prompt for Fine Location Permission*  
+Eligible users must allow fine location permission, which subsequently triggers app registration as described in the [Registering a User](https://github.com/placed/android-placed-sdk#registering-a-user) section.
+
+For an example of the opt-in dialog and location permission prompt, please refer to the [main activity of the sample app](https://github.com/placed/android-placed-sdk/blob/master/SampleApp/app/src/main/java/com/placed/android/sampleapp/MainActivity.java). We have also provided a [gallery for inspiration](https://github.com/placed/android-placed-sdk/blob/master/gallery) on how you can better integrate the opt-in experience into your app.
```

Comment:
Could we have this be a relative link? https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files